### PR TITLE
Misc. development, mostly recipes and vertical slabs.

### PR DESCRIPTION
**Pull request by FlenarnDev**

### CODE REVIEW

**Review by Dariensg — COMMENTED** [review]
LGTM, Swan will also review and then, if he approves it, it will be merged

File: src/main/java/com/nekomaster1000/infernalexp/init/IEBlocks.java [hunk 1]
```diff
@@ -1,21 +1,7 @@
...
-import com.nekomaster1000.infernalexp.blocks.GlowFireBlock;
-import com.nekomaster1000.infernalexp.blocks.GlowSandBlock;
-import com.nekomaster1000.infernalexp.blocks.GlowTorchBlock;
-import com.nekomaster1000.infernalexp.blocks.GlowWallTorchBlock;
-import com.nekomaster1000.infernalexp.blocks.GlowdustBlock;
-import com.nekomaster1000.infernalexp.blocks.LuminousFungusBlock;
-import com.nekomaster1000.infernalexp.blocks.ShroomlightFungusBlock;
-import com.nekomaster1000.infernalexp.blocks.SmoothGlowstonePressurePlateBlock;
-import com.nekomaster1000.infernalexp.blocks.TrappedGlowSandBlock;
+import com.nekomaster1000.infernalexp.blocks.*;
```
Comment (SwanX1) [inline src/main/java/com/nekomaster1000/infernalexp/init/IEBlocks.java]: revert this

File: src/main/java/com/nekomaster1000/infernalexp/init/IEBlocks.java [hunk 2]
```diff
@@ -25,6 +11,8 @@
 import net.minecraftforge.fml.RegistryObject;
 import net.minecraftforge.registries.DeferredRegister;
 import net.minecraftforge.registries.ForgeRegistries;
+import org.lwjgl.system.CallbackI;
+import org.omg.CORBA.PUBLIC_MEMBER;
```
Comment (SwanX1) [inline src/main/java/com/nekomaster1000/infernalexp/init/IEBlocks.java]:
```suggestion
```

**Review by SwanX1 — CHANGES_REQUESTED** [review]

File: src/main/java/com/nekomaster1000/infernalexp/init/IEBlocks.java [hunk 3]
```diff
@@ -1,7 +1,23 @@
 package com.nekomaster1000.infernalexp.init;
 
 import com.nekomaster1000.infernalexp.InfernalExpansion;
-import com.nekomaster1000.infernalexp.blocks.*;
+
```
Comment (SwanX1) [inline src/main/java/com/nekomaster1000/infernalexp/init/IEBlocks.java]:
```suggestion
```

File: src/main/java/com/nekomaster1000/infernalexp/init/IEBlocks.java [hunk 4]
```diff
@@ -11,8 +27,7 @@
 import net.minecraftforge.fml.RegistryObject;
 import net.minecraftforge.registries.DeferredRegister;
 import net.minecraftforge.registries.ForgeRegistries;
-import org.lwjgl.system.CallbackI;
-import org.omg.CORBA.PUBLIC_MEMBER;
+
```
Comment (SwanX1) [inline src/main/java/com/nekomaster1000/infernalexp/init/IEBlocks.java:29]:
```suggestion
```

**Review by SwanX1 — CHANGES_REQUESTED** [review]

**Pull request merged**